assets/scss/base/_table.scss

```diff
@@ -104,6 +104,11 @@ table {
     font-weight: 400;
   }
 }
+
+.table--heavy-header {
+    border-bottom: em(2) solid $govuk-blue-colour;
+}
```

Comment:
Given this is on the `th` element and there are no other header selectors I would be tempted to use:

`.table-header`

or

`.table__header`